.github/workflows/build.yaml

```diff
@@ -0,0 +1,18 @@
+name: Build project
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.23"
+
```

Comment:
isn't there already a CI job that does this?

Reply:
not that I can find